edb/pgsql/metaschema.py

```diff
@@ -5005,6 +5009,149 @@ def _generate_schema_alias_view(
     )
 
 
+def _generate_sql_information_schema() -> List[dbops.View]:
+    sql_ident = 'information_schema.sql_identifier'
+    sql_str = 'information_schema.character_data'
+    sql_bool = 'information_schema.yes_or_no'
+    sql_card = 'information_schema.cardinal_number'
+    tables_and_columns = [
+        dbops.View(
+            name=('edgedbsql', 'tables'),
+            query=(
+                f'''
+        SELECT
+            'postgres'::{sql_ident} AS table_catalog,
+            'public'::{sql_ident} AS table_schema,
+            substring(name from 10)::{sql_ident} AS table_name,
+            'BASE TABLE'::{sql_str} AS table_type,
+            NULL::{sql_ident} AS self_referencing_column_name,
+            NULL::{sql_str} AS reference_generation,
+            NULL::{sql_ident} AS user_defined_type_catalog,
+            NULL::{sql_ident} AS user_defined_type_schema,
+            NULL::{sql_ident} AS user_defined_type_name,
+            'NO'::{sql_bool} AS is_insertable_into,
+            'NO'::{sql_bool} AS is_typed,
+            NULL::{sql_str} AS commit_action
+        FROM edgedb."_SchemaObjectType"
+        WHERE name ilike 'default::%'
```

Comment:
Oh, we need to support non-default modules. We *could* filter out `std`/`schema`/etc but my inclination would be to leave them in?

Reply:
Yes indeed - I forgot we had modules. Can we map one module directly into schema name?

There will be a conflict if someone has a module named `information_schema`, but otherwise it should be ok.

Reply:
Yeah should be fine